From 8b2924af46b011c1502f2cde056e6aa05bb9ef14 Mon Sep 17 00:00:00 2001
From: Day Matchullis <dsm6069@gmail.com>
Date: Thu, 22 Oct 2026 20:21:22 -0600
Subject: [PATCH] Remove raylib arm64 binaries

Boulder picks them up as actual dependencies (even though they are not
actually used on an x64 machine)
---
 vendor/raylib/linux-arm64/libraylib.a      | 3 ---
 vendor/raylib/linux-arm64/libraylib.so.600 | 3 ---
 2 files changed, 6 deletions(-)
 delete mode 100644 vendor/raylib/linux-arm64/libraylib.a
 delete mode 100644 vendor/raylib/linux-arm64/libraylib.so.600

diff --git a/vendor/raylib/linux-arm64/libraylib.a b/vendor/raylib/linux-arm64/libraylib.a
deleted file mode 100644
index 0a8f83813..000000000
--- a/vendor/raylib/linux-arm64/libraylib.a
+++ /dev/null
@@ -1,3 +0,0 @@
-version https://git-lfs.github.com/spec/v1
-oid sha256:f35da654ad1f0223e833a76110d627659ab316a816dc770157c78bf18aeed65b
-size 2930116
diff --git a/vendor/raylib/linux-arm64/libraylib.so.600 b/vendor/raylib/linux-arm64/libraylib.so.600
deleted file mode 100644
index 457d5b749..000000000
--- a/vendor/raylib/linux-arm64/libraylib.so.600
+++ /dev/null
@@ -1,3 +0,0 @@
-version https://git-lfs.github.com/spec/v1
-oid sha256:af10d9d92cb18ae0770b0069a5d60d7bfd1da8f8d40bbf4ee8fed1d330028659
-size 2107056
-- 
2.56.0

